bin/rpm-s3

```diff
@@ -108,15 +108,20 @@ class FileGrabber(object):
 
 def getclient(base, host_url):
     if os.getenv('AWS_ACCESS_KEY'):
-        return boto.connect_s3(
+        client = boto.connect_s3(
             os.getenv('AWS_ACCESS_KEY'),
             os.getenv('AWS_SECRET_KEY'),
             host=host_url
-        ).get_bucket(base.netloc)
+        )
     else:
-        return boto.connect_s3(
+        client = boto.connect_s3(
             host=host_url
-        ).get_bucket(base.netloc)
+        )
+    b = client.get_bucket(options.bucket)
+    loc = b.get_location()  
+    if loc: # not US standard
```

Comment:
I don't think it's necessarily "not US standard", but you can achieve lower latencies if you use the DNS address of the bucket's region, instead of the generic s3.amazonaws.com that I _think_ works for all regions.

Reply:
Interesting. Before S3 was "global" (I'm thinking 2009-2011-ish) one could not find a bucket unless the client was connected to the specific region. I guess this is a good compromise (higher latency vs not finding the bucket at all and having to explicitly declare the region).